app/views/resources/index.html.erb

```diff
@@ -1,37 +1,88 @@
 <%= render "layouts/navbar", page: "Resources" %>
 
-<div class="container">
-    <h1 class="ui-shescoding-crud">Resources</h1>
-    <div class="col-sm-12">
-        <div class="row">
-            <div class="ui-shecodes-mission">
-              <table>
-                <thead>
-                  <tr>
-                    <th>Title</th>
-                    <th>Description</th>
-                    <th>Tags</th>
-                    <th colspan="3"></th>
-                  </tr>
-                </thead>
-
-                <tbody>
-                  <% @resources.each do |resource| %>
-                    <tr>
-                      <td><%= resource.title %> |</td>
-                      <td>| <%= resource.description %></td>
-                      <td>| <%= raw resource.tag_list.map { |t| link_to t, tag_path(t) }.join(', ') %></td>
-                      <td>| <%= link_to 'Show', resource %></td>
-                      <td>| <%= link_to 'Edit', edit_resource_path(resource) %></td>
-                      <td>| <%= link_to 'Destroy', resource, method: :delete, data: { confirm: 'Are you sure?' } %></td>
-                    </tr>
-                  <% end %>
-                </tbody>
-              </table>
-
-              <p><%= link_to 'New Resource', new_resource_path %></p>
-
+<div class="resources-section container">
+  <div class="row">
+    <div class="resources-sidebar col-md-3">
+      <ul>
+        <li class="selected" data-category="*"><%= link_to "All Resources", "" %></li>
```

Comment:
This needs to link to resources index, otherwise it will just do nothing. When you're filtering down on tags for example, you just stay on those tags instead of going to /resources.